db/migrate/20160510180328_add_small_biz_flag_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSmallBizFlagToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :small_business, :boolean, default: false, null: false
```

Comment:
Well, I don't know how much we will be querying on it, but it will be used to identify if the current_user can bid on it.